com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -58,6 +58,7 @@ internal class Styles
             // Post-processing
             public static GUIContent colorGradingMode = EditorGUIUtility.TrTextContent("Grading Mode", "Defines how color grading will be applied. Operators will react differently depending on the mode.");
             public static GUIContent colorGradingLutSize = EditorGUIUtility.TrTextContent("LUT size", "Sets the size of the internal and external color grading lookup textures (LUTs).");
+            public static GUIContent useFastSRGBLinearConversion = EditorGUIUtility.TrTextContent("Use Fast SRGB/Linear conversions", "Use faster, but less accurate approximation functions when converting to SRGB from Linear color space and vice versa.");
```

Comment:
Concerning the name of the property ("Use Fast SRGB/Linear conversions"): 
I'm not sure what's the current capitalization rule for UI elements. Please ask @TheoWong-pixel. It should either be "Use fast sRGB/Linear conversions" or "Use Fast sRGB/Linear Conversions".
sRGB is spelled with the lowercase "s".

The tooltip:
"Use faster, but less accurate approximation functions when converting between the sRGB and Linear color spaces."